backend/tracim_backend/migration/env.py

```diff
@@ -14,6 +16,11 @@
 # this is the Alembic Config object, which provides
 # access to the values within the .ini file in use.
 config = context.config
+# INFO - G.M - 2019-05-02 - hack to get env url instead of current config file
+# url for migration script
+env_sqlalchemy_url = os.environ.get("TRACIM_SQLALCHEMY__URL")
+if env_sqlalchemy_url:
+    config.set_main_option("sqlalchemy.url", env_sqlalchemy_url)
```

Comment:
Instead of 

```
env_sqlalchemy_url = os.environ.get("TRACIM_SQLALCHEMY__URL")
if env_sqlalchemy_url:
    config.set_main_option("sqlalchemy.url", env_sqlalchemy_url)
```

we should write this something like :

```
config.set_main_option("sqlalchemy.url", env_sqlalchemy_url, config.get_main_option("sqlalchemy.url")
```

no ?

Reply:
this syntax does not exist for config alembic file.

Reply:
ok.